packages/testing/src/fixture.ts

```diff
@@ -4,6 +4,7 @@ import { normalize, delimiter } from 'path'
 import { env } from 'process'
 import { fileURLToPath } from 'url'
 
+import { default as build, startDev } from '@netlify/build'
```

Comment:
Unrelated, hoisting the build import instead of asynchronously importing it as all fixtures use it anyhow